server/routes/api/moderator/moderatorHandler.ts

```diff
@@ -0,0 +1,25 @@
+import { Request, Response } from "express";
+
+import DBHandler from "./moderatorDBHandler";
+
+export default class Handler {
+  private dbHandler: DBHandler;
+
+  constructor() {
+    this.dbHandler = new DBHandler();
+  }
+
+  getDBHandler() {
+    return this.dbHandler;
+  }
+
+  getAllUsers = async (req: Request, res: Response) => {
+    try {
+      const data = await this.dbHandler.getAllUsers();
+      res.status(200).json({ message: "Successfully retrieved data", data });
```

Comment:
if you get null from db handler you'll still return 200. let's return an error code for that (this means no users in db 😆 )